Project.toml

```diff
@@ -41,6 +41,7 @@ julia = "1.6"
 [extras]
 ChainRulesCore = "d360d2e6-b24c-11e9-a2a3-2a2ae2dbcce4"
 DiffResults = "163ba53b-c6d8-5494-b064-1a9d43ac40c5"
+Documenter = "e30172f5-a6a5-5a46-863b-614d45cd2de4"
```

Comment:
Maybe it would be safer to add a compat entry for Documenter to Project.toml as well?

Reply:
I'm unsure about the role of compat entries for non-dependencies. Do they constrain package users who also happen to have Documenter in their environment, or just the version we use for testing AbstractDifferentiation?